en/django_start_project/README.md

```diff
@@ -118,6 +118,22 @@ ALLOWED_HOSTS = ['127.0.0.1', '.pythonanywhere.com']
 
 > Also add `.amazonaws.com` to the `ALLOWED_HOSTS` if you are using cloud9
 
+> If you are hosting your project on `Glitch.com`, let us protect the Django secret key that needs to 
+> remain confidential (otherwise, anyone remixing your project could see it) :
+>   * Create a file ```.env``` at the root of your project and add the following property in it :
+>     {% filename %}.env{% endfilename %}
+>     ```bash
+>     # Remember to change the value below to any, very long, very random sequence of characters
+>     SECRET='a-completely-random-string-of-letters-numbers-and-special-characters-anything-goes'
```

Comment:
Humans are bad at coming up with truly random stuff. Should we instruct participants to run
```
python -c 'from django.core.management.utils import get_random_secret_key; \
            print(get_random_secret_key())'
```
in the Glitch.com terminal? (command from [here](https://saasitive.com/tutorial/generate-django-secret-key/))

That works in the Glitch.com termianl after the setup instructions added by this pull request.

Reply:
Thanks for the tip, I updated and added the instructions to generate the secret key.